jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/impl/admin/UserTaskAdminServiceImplTest.java

```diff
@@ -161,6 +165,72 @@ public void testAddPotentialOwnersNotBusinessAdmin() {
         
     }
 
+    @Test(expected = TaskNotFoundException.class)
+    public void testAddPotentialOwnersToNonExistentTask() {
+        processInstanceId = processService.startProcess(deploymentUnit.getIdentifier(), "org.jbpm.writedocument");
+        assertNotNull(processInstanceId);
+
+        List<TaskSummary> tasks = runtimeDataService.getTasksAssignedAsPotentialOwner("salaboy", new QueryFilter());
+        assertEquals(1, tasks.size());
+        TaskSummary task = tasks.get(0);
+
+        userTaskService.release(task.getId(), "salaboy");
+        userTaskAdminService.addPotentialOwners(15456, false, factory.newUser("john"));
```

Comment:
I'd rather we use `Assertions` methods to catch the exception from a specific place, as opposed to catching the exception from anywhere at all using `@Test`.